### ISSUE TITLE
Support for MySQL

### ISSUE THREAD

**Issue by cvburgess** [cvburgess/SQLDataSource#23]
@cvburgess I use MySQL as the DB.  This is [knex raw](https://github.com/tgriesser/knex/issues/1802) method feature.

_Originally posted by @a2exfr in https://github.com/cvburgess/SQLDataSource/issues/19#issuecomment-489920171_

**Comment by a2exfr** [comment]
Hi. This is the way I get batched  normalized output not raw. What you think?

```
import DataLoader from 'dataloader';


export default class Dataloader {
	constructor(knex) {
		this.db = knex;
		this.queries = {};
		this.loader = new DataLoader(rawQueries =>
			Promise.all(rawQueries.map((rawQuery) => this.queries[rawQuery])),
		);
	}

	normalizeDBResult(result) {
		switch (this.db.client.config.client) {
			case 'postgres':
				return result && result.rows;
			case 'mssql':
				return result;
			case 'sqlite3':
				return result;
			case 'mysql':
				return result;
			case 'mysql2':
			case 'oracledb':
			case 'redshift':
			default:
				return result;
		}
	}

	getBatched(query) {
		const queryString = query.toString();
		this.queries[queryString] = query;
		return this.loader.load(queryString).then(result => this.normalizeDBResult(result));
	}

}
```

**Comment by cvburgess** [comment]
Hey there thank you for the help!

Would you mind opening this as a PR so i can more easily see the diff and test it locally?

**Comment by cvburgess** [comment]
Fixed in version 1.0! 🎉
